docs/how-to-guides/new-user-guides/kubernetes-cluster-setup/k3s-for-rancher.md

```diff
@@ -45,7 +45,17 @@ When running the command to start the K3s Kubernetes API server, you will pass i
 
   :::
 
-1. Repeat the same command on your second K3s server node.
+1. Get main server node token:
+  ```
+  cat /var/lib/rancher/k3s/server/node-token
```

Comment:
```suggestion
  cat /var/lib/rancher/k3s/server/token
```

Looking at the [K3s docs](https://docs.k3s.io/datastore/ha#3-optional-join-additional-server-nodes), it shows `token` instead of `node-token`. @alex1704 are you seeing `node-token` in your setup?

Reply:
@btat I see both and they are equal.